docs/theme/generate-themes.ts

```diff
@@ -134,7 +134,96 @@ In addition to themes and theme modifiers, there are special aliases:
 - \`light\` - an alias for \`${light}\`
 - \`dark\` - an alias for \`${dark}\`
 
-On its own, \`default\` is equivalent to \`[light, dark]\` (or \`[${light}, ${dark}]\`). The \`default\` theme is applied by default if you don’t specify any color theme. You can also use \`default\` to combine a specific light or dark theme with the default theme of the opposing mode; for example \`[cotton, default]\` is equivalent to \`[cotton, dark]\`, and \`[coffee, default]\` is equivalent to \`[coffee, light]\`.`;
+On its own, \`default\` is equivalent to \`[light, dark]\` (or \`[${light}, ${dark}]\`). The \`default\` theme is applied by default if you don’t specify any color theme. You can also use \`default\` to combine a specific light or dark theme with the default theme of the opposing mode; for example \`[cotton, default]\` is equivalent to \`[cotton, dark]\`, and \`[coffee, default]\` is equivalent to \`[coffee, light]\`.
```

Comment:
I think this page should probably at least mention how to create your own theme.

Reply:
It’s mentioned right at the top; the answer is to create a [custom stylesheet](https://observablehq.com/framework/config#style).